<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Settings</class>
    <widget class="QDialog" name="Settings">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>803</width>
                <height>466</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>OnexExplorer Settings</string>
        </property>
        <property name="windowIcon">
            <iconset resource="../../resources.qrc">
                <normaloff>:/resources/oxe_icon_trans.ico</normaloff>:/resources/oxe_icon_trans.ico
            </iconset>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="1" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <spacer name="horizontalSpacer_2">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>10</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_replace">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>40</width>
                                    <height>40</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>16</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>...</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_title_replace">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>16777215</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>14</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Default Replace Path:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_replace">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>16777215</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>10</pointsize>
                                    <weight>50</weight>
                                    <bold>false</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="1" column="1" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>10</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_export">
                            <property name="maximumSize">
                                <size>
                                    <width>40</width>
                                    <height>40</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>16</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>...</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_title_export">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>16777215</width>
                                    <height>40</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>14</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Default Export Path:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_export">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>16777215</width>
                                    <height>40</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>10</pointsize>
                                    <weight>50</weight>
                                    <bold>false</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="5" column="2">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
            <item row="3" column="1" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                        <spacer name="horizontalSpacer_3">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>10</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_open">
                            <property name="maximumSize">
                                <size>
                                    <width>40</width>
                                    <height>40</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>16</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>...</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_title_open">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>14</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Default .NOS Open Path:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_open">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>10</pointsize>
                                    <weight>50</weight>
                                    <bold>false</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="4" column="1" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                        <spacer name="horizontalSpacer_4">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Fixed</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>10</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_save">
                            <property name="maximumSize">
                                <size>
                                    <width>40</width>
                                    <height>40</height>
                                </size>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>16</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>...</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_title_save">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>14</pointsize>
                                </font>
                            </property>
                            <property name="text">
                                <string>Default .NOS Save Path:</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLabel" name="label_save">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>10</pointsize>
                                    <weight>50</weight>
                                    <bold>false</bold>
                                </font>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item row="5" column="1">
                <widget class="QPushButton" name="btn_reset">
                    <property name="text">
                        <string>Reset</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="../../resources.qrc"/>
    </resources>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>Settings</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>Settings</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
